node/src/BaseClient.ts

```diff
@@ -2125,12 +2125,24 @@ export class BaseClient {
      *
      * @param keys_and_ids - pairs of keys and entry ids to read from. A pair is composed of a stream's key and the id of the entry after which the stream will be read.
      * @param options - options detailing how to read the stream.
-     * @returns A map between a stream key, and an array of entries in the matching key. The entries are in an [id, fields[]] format.
+     * @returns A map of stream keys, to a map of stream ids, to an array of entries.
+     * @example
+     * ```typescript
+     * const streamResults = await client.xread({"my_stream": "0-0", "writers": "0-0"});
+     * console.log(result); // Output: {
+     *                      //     "my_stream": {
+     *                      //         "1526984818136-0": ["duration", "1532", "event-id", "5", "user-id", "7782813"],
+     *                      //         "1526999352406-0": ["duration", "812", "event-id", "9", "user-id", "388234"],
+     *                      //     }, "writers": {
+     *                      //         "1526985676425-0": ["name", "Virginia", "surname", "Woolf"],
+     *                      //         "1526985685298-0": ["name", "Jane", "surname", "Austen"],
+     *                      //     },
```

Comment:
Maybe remove indentation for entire block
```suggestion
     *                      //     }
     *                      //   }
```